frontend/src/pages/admin/AdminRegistrationPage.js

```diff
@@ -0,0 +1,9 @@
+import AdminRegistrationForm from "../../components/admin/AdminRegistrationForm";
```

Comment:
Admin dont need registration for security purposes. Create a user via seeder

Reply:
Ahhhh okay2 sir 🙇

Reply:
There will be no more Registration page and Registration Admin route. 🙇 
![image](https://user-images.githubusercontent.com/110363852/212280239-05bf3c5b-0eb5-48a6-bfb3-f29d9cbda3b6.png)
![image](https://user-images.githubusercontent.com/110363852/212280493-cda36811-5ea9-48ca-b349-7430485c0458.png)
![image](https://user-images.githubusercontent.com/110363852/212280625-e3cb7546-2a0f-4ad5-9137-9a541caaa75c.png)